infra/core/search/search-services.bicep

```diff
@@ -8,6 +8,7 @@ param sku object = {
 
 param authOptions object = {}
 param semanticSearch string = 'disabled'
+param searchApiKeyEnabled bool = true
```

Comment:
cc @jongio who likes to be notified about potential changes to infra/core Bicep files (as we try to reuse the same infra/core across azd templates)